jax/scipy/optimize/bfgs_minimize.py

```diff
@@ -0,0 +1,127 @@
+"""The Broyden-Fletcher-Goldfarb-Shanno minimization algorithm.
+https://pages.mtu.edu/~struther/Courses/OLD/Sp2013/5630/Jorge_Nocedal_Numerical_optimization_267490.pdf
+"""
+
+import jax
+import jax.numpy as np
+from jax.lax import while_loop
+from .line_search import line_search
+import collections
+
+BFGSResults = collections.namedtuple(
```

Comment:
[Optional] consider using ``typing.NamedTuple`` to track types in your named tuple.

Reply:
Done.